### ISSUE TITLE
Flipbox\LumenGenerator\Console\ConsoleMakeCommand::handle() does not exist

### ISSUE THREAD

**Issue by semsphy** [flipboxstudio/lumen-generator#18]
I got reception in latest  version of Lumen (5.5.0) (Laravel Components 5.5.*).

**Comment by typoworx-de** [comment]
I can provide a really simple hotfix!
https://github.com/typoworx-de/lumen-generator/tree/hotfix-issue-18

We just need an Abstract Class providing a shorthand to all Command-Classes and replace "extends Command" with "AbstractCommand". For a ready-patched version look at my fork.

Please approve and merge into this project.

```
<?php
namespace Flipbox\LumenGenerator\Console;

use Illuminate\Console\Command;

class AbstractCommand extends Command
{
    /**
     * Shorthand for Laravel 5.5.*
     * https://github.com/flipboxstudio/lumen-generator/issues/18
     *
     * @return mixed
     */
    public function handle()
    {
        $this->fire();
    }
}
```

**Comment by barryrld** [comment]
Possible duplicate with this PR? : https://github.com/flipboxstudio/lumen-generator/pull/17

**Comment by joelhy** [comment]
Fixed by PR #17
